### DISCUSSION TITLE
Umbraco forms start folder at group level

### DISCUSSION THREAD

**Discussion by bjornnorlen** [umbraco/Umbraco-CMS#12145] [Features and ideas]
Would it be possible to add support for forms start folder in the Group permissions tree in the users section? If doing so, all users belonging to that user group will have that folder as a start node. As it is right now you need to set the startnode on each individual user, which btw is great, but it might be a bit time consuming if there are many users, all of which are going to have the same start node if they belong to the same user group. We are building a solution which will host several different websites and each website will have its own user groups(s) with different users. It would have been awesome if you then just could select forms start node for the entire group instead of each individual user.

**Comment by nul800sebastiaan** [comment]
Make sure to create an issue for this on https://github.com/umbraco/Umbraco.Forms.Issues - don't think the Forms teams monitors the CMS discussions 🙂

**Reply by bjornnorlen** [reply]
Thanks, will do :)
